locations/spiders/call_a_pizza_de.py

```diff
@@ -0,0 +1,73 @@
+import json
+import re
+
+from scrapy.spiders import Spider
+
+from locations import hours
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class CallAPizzaDESpider(Spider):
+    name = "call_a_pizza_de"
+    item_attributes = {"brand": "Call a Pizza", "brand_wikidata": "Q1027107"}
+    start_urls = ["https://www.call-a-pizza.de/bestellen"]
+    DAYS_DE = ["Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag", "Sonntag"]
+    DAYS2EN = dict(zip(DAYS_DE, hours.DAYS))
```

Comment:
Take a look in the [`locations/hours.py` file](https://github.com/alltheplaces/alltheplaces/blob/master/locations/hours.py). There are some [constants](https://github.com/alltheplaces/alltheplaces/blob/master/locations/hours.py#L63-L79) that already handle this mapping.